packages/jarm/lib/jarm-auth-response/v-jarm-auth-response-params.ts

```diff
@@ -0,0 +1,62 @@
+import { checkExp } from '@sphereon/oid4vc-common';
+import * as v from 'valibot';
+
+export const vJarmAuthResponseErrorParams = v.looseObject({
+  error: v.string(),
+  state: v.optional(v.string()),
+
+  error_description: v.pipe(
+    v.optional(v.string()),
+    v.description('Text providing additional information, used to assist the client developer in understanding the error that occurred.'),
```

Comment:
Are these replaced by actual descriptions, otherwise it needs to be updated

Reply:
I am not entirely sure what you mean by actual descriptions. 
But the descriptions are not shown to the user if that is what you mean. It's more about attaching them as metadata to the fields.

Reply:
Okay, probably should look into valibot. The text suggests an example from their repo; I would expect the description to be not so generic